spread-tests/main/core-is-preferred/task.yaml

```diff
@@ -0,0 +1,10 @@
+summary: The snap named 'core' is preferred to the snap 'ubuntu-core'
+prepare: |
+    snap install --devmode snapd-hacker-toolbelt
+    snap install core
```

Comment:
The "ubuntu-core" snap is installed early in spread-prepare.sh. Even if installing snapd-hacker-toolbelt would pull in core we still install it separately. I think this is correct.